### PR TITLE
Create oliik2013.thedev.me.json

### PR DESCRIPTION
**Pull request by oliik2013**
yay a domain for me

### CODE REVIEW

**Comment by oliik2013** [comment]
fixed the json syntax

File: domains/oliik2013.thedev.me.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "description": "domain for me yay for now for website",
 
   "records": {
-    "CNAME": "oliik2013.github.io"
+    "CNAME": [oliik2013.github.io]
```
Comment (YazeedT) [inline domains/oliik2013.thedev.me.json]: Add it as a string i.e. ["oliik2013.github.io"]

Comment (oliik2013) [reply domains/oliik2013.thedev.me.json]: i changed it @YazeedT

**Comment by oliik2013** [comment]
fix json again @YazeedT

**Pull request merged**